src/main/java/org/owasp/html/HtmlEntities.java

```diff
@@ -28,13 +28,27 @@
 
 package org.owasp.html;
 
+import java.util.Map;
+
 import com.google.common.collect.ImmutableMap;
 
 /**
  * Utilities for decoding HTML entities, e.g., {@code &amp;}.
  */
 final class HtmlEntities {
 
+  private static final int LONGEST_ENTITY_NAME;
+
+  static {
+    int longestEntityName = 0;
+    for (String entityName : getEntityNameToCodePointMap().keySet()) {
+      if (entityName.length() > longestEntityName) {
+        longestEntityName = entityName.length();
+      }
+    }
+    LONGEST_ENTITY_NAME = longestEntityName;
```

Comment:
Since I had to factor out the generation of the map to a separate method anyway, I decided to dynamically calculate this value. However if you don't like the static initializer we could hardcode it as well. The longest entity name is CounterClockwiseContourIntegral, which is 31 characters.